### ISSUE TITLE
{Linked:Bug41431272; LinkedBug}[Voice access - Adaptive Cards]: User is not able to access 'Authcardbutton' link Using 'Click and Move to' Commands.

### ISSUE THREAD

**Issue by Susmitha474** [microsoft/AdaptiveCards#7929]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

Test Environment:

OS Build: 25201.1000
Edge dev: Version 107.0.1379.1 (official build) dev (64-bit)

Application URL: [Schema Explorer | Adaptive Cards](https://adaptivecards.io/explorer/)

Accessibility Tool: Voice Access

Pre-requisite:
OS Settings->Accessibility->Voice Access (Turn ON)

Repro Steps:
Launch the above-mentioned URL on edge Dev with Valid credentials.
Command: "Voice access wake up"
Command: "Click Authcardbutton".
Command: "Move to Authcardbutton".
Observe the behavior.
Actual Result:
User is unable to access the "Authcardbutton" using Click/Move To Command.

Expected Result:
User Should be able to access the "Authcardbutton" link using "Click/Move to" VA commands.

Note:
same issue repro for Action.OpenUrl link.
PFA: Code snippet for Action.openUrl.png
Same issue repro at "Action.OpenUrl" button present under the Adaptive cards.
PFA: open url issue.mp4

Observation:
unable to access the controls using click/move to wherever 
1.Special Characters: controls are having special characters like action.Choiceset.
2.text Without Space: ex: "backgroundimage" VA is taking it as Background image, so VA is not able to recognize it, as space is included between these 2words.
3.finally, user is able to access it by conveying with Spelling(Which is not a good Practice).

PFA: Backgroundimage.mp4

User Experience:
Voice access users will get impact and will get disoriented if this behavior persists.

MAS Reference:
[No Disruption of Accessibility Features - Liquid (microsoft.com)](https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/04.03.01)

Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)"

### Screenshots

https://user-images.githubusercontent.com/104482612/192774923-82600e8c-e6c4-4013-91c1-333708e9eb86.mp4

<img width="960" alt="Code snippet for Authcardbutton" src="https://user-images.githubusercontent.com/104482612/192774493-798b2163-ef07-48fa-a637-fae81a252cb5.png">

https://user-images.githubusercontent.com/104482612/192774573-f2ced74c-68cc-4026-9d4b-b945d0e776a8.mp4

https://user-images.githubusercontent.com/104482612/192774240-a1ac50f4-4d56-4889-8f6f-bb701c015fc7.mp4

<img width="960" alt="Code snippet for Action openUrl" src="https://user-images.githubusercontent.com/104482612/192774274-96b3cef0-7d9f-4227-96a8-f9b8b18b8fa6.png">

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
As per the comment mentioned in the partner bug, closing this issue.

**Test Environment:**
URL: [Schema Explorer | Adaptive Cards](https://adaptivecards.io/explorer/Action.OpenUrl.html)
OS Version:  22H2(OS Build 25276.1000)
Browser Version: Version 111.0.1619.2 (Official build) dev (64-bit)

![Issue is not repro on show number (1)](https://user-images.githubusercontent.com/95913748/214309304-c9af8ba9-5d68-49a1-bfca-5f901fa427cd.png)
